docs/modules/ROOT/pages/getting-spring-security.adoc

```diff
@@ -138,12 +243,24 @@ This approach uses Maven's "`bill of materials`" (BOM) concept and is only avail
 For additional details about how dependencies are resolved, see https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html[Maven's Introduction to the Dependency Mechanism documentation].
 ====
 
-[[maven-repositories]]
-=== Maven Repositories
-All GA releases (that is, versions ending in .RELEASE) are deployed to Maven Central, so you need not declare additional Maven repositories in your pom.
+[[maven-gradle-gearepositories]]
+=== Maven and Gradle Repositories
+All GA releases (that is, versions ending in .RELEASE) are deployed to Maven Central, so you need not declare additional Maven repositories in your `pom.xml` or, for Gradle, using the `mavenCentral()` repository is sufficient for GA releases.
```

Comment:
Please place the "For Gradle" in a separate sentence and separate paragraph. This makes the sentence shorter and continue to be about one thing.